src/main/java/com/prgrms/springbootjpa/domain/order/Member.java

```diff
@@ -0,0 +1,40 @@
+package com.prgrms.springbootjpa.domain.order;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "member")
+public class Member extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "name", nullable = false, length = 30)
+    private String name;
+
+    @Column(nullable = false, length = 30, unique = true)
```

Comment:
```suggestion
    @Column(nullable = false, length = 30, unique = true, name = "nick_name")
```
컬럼과 객체를 매핑할 때 각 컨벤션을 맞추면 가독성이 좋아질 것 같아요!